pkg/sethandler/controller.go

```diff
@@ -284,9 +285,23 @@ func determineCid(podStatus v1.PodStatus, containerName string) string {
 	return ""
 }
 
+func trimContainerPrefix(contName string) (string, error) {
+	for _, prefix := range containerPrefixList{
+		if strings.HasPrefix(contName, prefix){
+			return strings.TrimPrefix(contName, prefix), nil
+		}
+	}
+	return "",  fmt.Errorf("container name does not contain predefined prefixes (%s) container name: %s",containerPrefixList, contName)
```

Comment:
this is not necessairly an error. if no prefix it just means some runtimes already created them "properly"

Reply:
altho you wont have an error returned after modifying the signature, so just remove :)